src/index.html

```diff
@@ -1,13 +1,313 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="./images/favicon.svg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo">
+              <img
+                src="./images/logo-white.svg"
+                alt="MyBike"
+                class="top-bar__logo-image"
+              >
+            </a>
+            <div class="top-bar__icons">
+              <a href="tel:+1 234 555-55-55" class="icon icon--phone">
+                <div class="icon__tooltip">
+                  +1 234 555-55-55
+                </div>
+              </a>
+
+              <a href="#menu" class="icon icon--menu"></a>
+
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">Take the Streets</h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo">
+            <img
+            src="./images/logo-black.svg"
+            alt="MyBike"
+            class="top-bar__logo-image"
+            >
```

Comment:
Need to fix and check it everywhere
```suggestion
            <img
              src="./images/logo-black.svg"
              alt="MyBike"
              class="top-bar__logo-image"
            >
```